### ISSUE TITLE
Unable to load shared library 'SDL2' or one of its dependencies.

### ISSUE THREAD

**Issue by se5a** [Pulsar4xDevs/Pulsar4x#307]
I've recently updated to a fresh install of linux mint 20.3
however I'm now having problems with SDL2. 
I've manually installed SDL2.2.0.20 without luck, still looking into what's going on.

**Comment by se5a** [comment]
It might be worth looking at including the steam runtime to help mitigate this problem. 
I have looked into this briefly but it's not particularly well explained.

**Comment by se5a** [comment]
```
$ LD_DEBUG=libs "dotnet watch" 2>&1 >/dev/null | grep trying
     77849:	  trying file=/lib/x86_64-linux-gnu/libc.so.6
     77849:	  trying file=/lib/x86_64-linux-gnu/libpthread.so.0
     77849:	  trying file=/lib/x86_64-linux-gnu/libdl.so.2
     77849:	  trying file=/lib/x86_64-linux-gnu/libutil.so.1
     77849:	  trying file=/lib/x86_64-linux-gnu/libm.so.6
     77849:	  trying file=/lib/x86_64-linux-gnu/libexpat.so.1
     77849:	  trying file=/lib/x86_64-linux-gnu/libz.so.1
     77849:	  trying file=/lib/x86_64-linux-gnu/libsqlite3.so.0
     77849:	  trying file=/lib/x86_64-linux-gnu/libapt-pkg.so.6.0
     77849:	  trying file=/lib/x86_64-linux-gnu/libstdc++.so.6
     77849:	  trying file=/lib/x86_64-linux-gnu/libgcc_s.so.1
     77849:	  trying file=/lib/x86_64-linux-gnu/libresolv.so.2
     77849:	  trying file=/lib/x86_64-linux-gnu/libbz2.so.1.0
     77849:	  trying file=/lib/x86_64-linux-gnu/liblzma.so.5
     77849:	  trying file=/lib/x86_64-linux-gnu/liblz4.so.1
     77849:	  trying file=/lib/x86_64-linux-gnu/libzstd.so.1
     77849:	  trying file=/lib/x86_64-linux-gnu/libudev.so.1
     77849:	  trying file=/lib/x86_64-linux-gnu/libsystemd.so.0
     77849:	  trying file=/lib/x86_64-linux-gnu/libgcrypt.so.20
     77849:	  trying file=/lib/x86_64-linux-gnu/librt.so.1
     77849:	  trying file=/lib/x86_64-linux-gnu/libgpg-error.so.0
     77849:	  trying file=/lib/x86_64-linux-gnu/libnss_files.so.2
```
Does not apear to find any of the DSL2 libraries. not sure whats going on here. 


on the steam runtime front, these blog posts could be useful:
https://jorgen.tjer.no/post/2014/05/26/self-contained-game-distribution-on-linux/
https://jorgen.tjer.no/post/2014/05/28/steam-runtime-without-steam/

It'd add about 100mb to the game which I'm not terribly keen on, but may be worth it.  
I'm out of my depth trying to figure out exactly what I'm supposed to do to get it going in a .net environment.
maybe figuring out how to do *just* SDL2 would be better (I think Dwarf Fortress does this?)

**Comment by se5a** [comment]
Ended up being that I'd not correctly installed SDL2 correctly. 
thanks for Myvar for taking a look and figuring it out. 
we also found another issue where I'm setting the version of OpenGL which is unnecessary so I've commented that out in the code. 

Notes for others coming across this:
 - ensure you're in the correct directory when you `sudo make install`.
 - sudo make install: 
   - SDL2, 
   - SDL2_image, 
   - SDL2_mixer, and 
   - SDL2_ttf 

https://wiki.libsdl.org/Installation
